apps/meteor/server/models/raw/LivechatInquiry.ts

```diff
@@ -91,4 +92,47 @@ export class LivechatInquiryRaw extends BaseRaw<ILivechatInquiryRecord> implemen
 			{ $unset: { locked: 1, lockedAt: 1 } },
 		);
 	}
+
+	async getSortingQuery(sort: Record<string, 1 | -1>): Promise<Document[]> {
```

Comment:
WDYT of returning the sort instead of modify it in place? And then do something like:
```
{ ...currentSort, ...getSortingQuery() }
```
That way, you don't have functions with sideeffects on models

Reply:
makes sense.